internal/cli/atlas/users/invite.go

```diff
@@ -147,27 +147,28 @@ func splitRole(role string) ([]string, error) {
 	return value, nil
 }
 
-func newAtlasProjectRole(role string) (atlasv2.RoleAssignment, error) {
+func newAtlasProjectRole(role string) (atlasv2.CloudRoleAssignment, error) {
 	value, err := splitRole(role)
 	if err != nil {
-		return atlasv2.RoleAssignment{}, err
+		return atlasv2.CloudRoleAssignment{}, err
 	}
-	atlasRole := atlasv2.RoleAssignment{
-		GroupId: &value[0],
-		Role:    pointer.Get(strings.ToUpper(value[1])),
+	atlasRole := atlasv2.CloudRoleAssignment{
+		GroupId: &value[0], 
+		// TODO fix this
+		RoleName:    pointer.Get(strings.ToUpper(value[1])),
```

Comment:
Notice that this is renamed from `RoleName` from `Role`. 
Would that be correct?